.github/workflows/py-dependency-check.yml

```diff
@@ -0,0 +1,76 @@
+name: Python dependency checks
+
+# This workflow is intended to check that the latest release of the `cellxgene-census` package
+# continues to function correctly using the latest upstream dependencies.
+#
+# Method:
+# 1. Check out the repo at the latest release tag
+# 2. Install latest cellxgene-census package from pypi
+# 3. Verify versions match
+# 4. Run tests from the repo
+
+on:
+  schedule:
```

Comment:
do we want this to run on any other events?

if not, how will we monitor that it is working?  Badge on the REAMDE?

Reply:
@pablo-gar - you can make the call. Either in this PR or elsewhere.  This is not the only workflow that needs such a mechanism.

Reply:
Unless any strong concerns let's go with slack since that's already a convention used by the team

Reply:
I am unable to figure out how this works with a quick review, so I'm putting this PR on a back-burner for a while.

Reply:
added badge to README.md